ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -544,7 +544,7 @@ if get_option('enable-mediapipe')
     join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/com_google_absl'),
   )
 
-  cmd = run_command('sh', '-c', meson.source_root() + '/tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir)
+  cmd = run_command('sh', '-c', meson.source_root() + '/tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir, check : true)
   if cmd.returncode() != 0
     error(cmd.stderr().strip())
```

Comment:
You can now drop this error message, and I don't think you need to save the return value to `cmd`.

The `check: true` change means that Meson now errors out for you.

Reply:
I agree.

Reply:
Indeed. The if block to test ```cmd.returncode()``` has been removed.